internal/controller/shipwrightbuild_controller.go

```diff
@@ -59,6 +60,20 @@ func (r *ShipwrightBuildReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		return err
 	}
 
+	// Remove runAsUser and runAsGroup from a Deployment container's security context
+	// Insert Openshift Service CA annotations in service and CRD
+	if r.Manifest, err = r.Manifest.Transform(
+		common.RemoveRunAsUserRunAsGroup,
+		common.InjectAnnotations("Service", common.ShipwrightWebhookServiceName, map[string]string{
+			openshiftserviceca.ServingCertSecretAnnotation: common.ShipwrightWebhookCertSecretName,
+		}),
+		common.InjectAnnotations("CustomResourceDefinition", "", map[string]string{
```

Comment:
Does this add the annotation to _all_ CRD instances in the manifest?

Thinking long-term: we probably want to be specific about which CRDs require the conversion webhook. Looking at Tekton as an example, some APIs have graduated to `v1` while others still have `v1beta1` and don't have conversion webhooks for prior `alpha` versions. We may find ourselves in a similar situation with Shipwright.

Reply:
Yes. This would add to all the CRDs, but in future we can pass arrays in `kind` and `name` to have more specific targets.